lib/src/theme/flutter_deck_theme.dart

```diff
@@ -106,6 +106,10 @@ class FlutterDeckThemeData {
         subtitleTextStyle: textTheme.subtitle,
         titleTextStyle: textTheme.title,
       ),
+      bigFactSlideTheme: FlutterDeckBigFactSlideThemeData(
+        titleTextStyle: textTheme.display,
+        subtitleTextStyle: textTheme.subtitle,
+      ),
```

Comment:
I think we need to order this alphabetically as well ? @mkobuolys

Reply:
Yup, would be nice as well ✌️